app/core/output/__init__.py

```diff
@@ -27,7 +26,7 @@ def print_program_from_provider(filename=os.path.join('data', 'GN-2018B-Q-101.js
         data = json.loads(f.read())
 
     program = provider.parse_program(data['PROGRAM_BASIC'])
-    print_program(program)
+    program.show()
```

Comment:
Annoying changes, am I right?

Reply:
Annoying how? I think you moving `show` into `Program` is fine.

Reply:
lol just a joke because was my change